### PR TITLE
Prevent selection of current docket on DocketSwitchReviewRequestForm

### PR DESCRIPTION
**Pull request by jcq**
### Description
Adds logic to `DocketSwitchReviewRequestForm` to prevent selection of the current docket type

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] Radio button for current docket type is disabled and label indicates it is current type

### Design
Screen Shot 2021-02-05 at 12 47 24 PM

### Testing
Verification of this can be done in Storybook

### CODE REVIEW

**Comment by jcq** [comment]
## Updated Screenshot

Screen Shot 2021-02-05 at 16 13 42

File: client/test/app/queue/docketSwitch/DocketSwitchReviewRequestForm.test.js [hunk 1]
```diff
@@ -40,6 +41,24 @@ describe('DocketSwitchReviewRequestForm', () => {
...
 
+  it('disables current docket', async () => {
+    render(<DocketSwitchReviewRequestForm {...defaults} />);
+
+    // Set disposition to show docket selection
+    await userEvent.click(
+      screen.getByRole('radio', { name: /grant all issues/i })
+    );
+
+    await waitFor(() => {
```
Comment (leikkisa) [inline client/test/app/queue/docketSwitch/DocketSwitchReviewRequestForm.test.js:52]: Nice

**Review by leikkisa — APPROVED** [review]
Nicely done, glad you and Rutvi caught this, it's a good improvement for the user.

**Review by leikkisa — APPROVED** [review]
Oops, wrong PR - redacting/moving my review comment.

**Comment by rutvigupta-design** [comment]
Thanks for catching @jcq and putting out such a quick fix! 🎉

**Pull request merged**